api/src/main/java/org/pmiops/workbench/captcha/CaptchaVerificationServiceImpl.java

```diff
@@ -17,7 +21,21 @@
 
   final String urlPattern = "https://%s/login";
 
-  final String localHostUrlPattern = "http://%s:4200/login";
+  final List<String> nonTestCaptchaURLs =
+      new ArrayList<String>(
+          Arrays.asList(
+              "https://workbench.researchallofus.org/login",
+              "https://all-of-us-rw-stable.appspot.com/login"));
+
+  final List<String> testCaptchaURLs =
+      new ArrayList<String>(
+          Arrays.asList(
+              "http://localhost:4200/login",
+              "https://all-of-us-workbench-test.appspot.com",
+              "https://all-of-us-rw-staging.appspot.com/login",
+              "https://all-of-us-rw-perf.appspot.com/login"));
+
+  final String googleTestHost = "testkey.google.com";
```

Comment:
Could you add some comments that indicate what the behavior is for a "test captcha"? Are these just two separate keys,  or does it just allow any captcha input to pass?